website/sidebars.json

```diff
@@ -133,6 +133,7 @@
       "example-findByText",
       "example-input-event",
       "example-react-context",
+      "example-react-formik",
```

Comment:
If we move this after`useReducer`, the React core examples will be grouped before the third party library examples.